src/main/java/org/apache/sling/extensions/webconsolesecurityprovider/internal/ServicesListener.java

```diff
@@ -80,25 +84,27 @@ public ServicesListener(final BundleContext bundleContext) {
         this.authSupportListener.start();
         this.repositoryListener.start();
         this.authListener.start();
+        forceJcrAuth = bundleContext.getProperty(WEBCONSOLE_FORCE_AUTH_AGAINST_JCR) != null;
```

Comment:
Instead of a boolean, a string value could be used with the values "JCR" and "SLING", to select between the two providers